src/pt/xxxyaoi/src/eu/kanade/tachiyomi/extension/pt/xxxyaoi/XXXYaoi.kt

```diff
@@ -0,0 +1,24 @@
+package eu.kanade.tachiyomi.extension.pt.xxxyaoi
+
+import eu.kanade.tachiyomi.multisrc.madara.Madara
+import eu.kanade.tachiyomi.network.interceptor.rateLimit
+import okhttp3.OkHttpClient
+import java.text.SimpleDateFormat
+import java.util.Locale
+import java.util.concurrent.TimeUnit
+
+class XXXYasooi : Madara(
```

Comment:
class must have the same name as the file

Reply:
Didn't see it before mb
I just fixed it